settings.gradle.kts

```diff
@@ -6,6 +6,7 @@ pluginManagement {
   }
 }
 
-include(":app", ":sdk", ":extension-style", ":plugin-overlay", ":plugin-compass", ":plugin-logo", ":plugin-scalebar", ":plugin-gestures", ":plugin-attribution", ":plugin-annotation", ":plugin-locationcomponent", ":plugin-animation", ":module-telemetry", ":sdk-base", ":extension-style-app")
+include(":app", ":sdk", ":extension-style", ":plugin-overlay",
```

Comment:
nit: can we keep it in single line? I'm afraid this might break our parser in the binary size metrics collection.